components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/graphQL/GraphQLAPIHandler.java

```diff
@@ -58,21 +52,20 @@
 import java.util.Base64;
 import java.net.URLDecoder;
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Map;
 import java.util.Set;
 import javax.xml.stream.XMLStreamException;
 
+import static org.apache.axis2.Constants.Configuration.*;
```

Comment:
avoid multi imports.

Reply:
We can't avoid multi import in constants because here we used Constants.Configuration.HTTP_METHOD in twice.